src/System/View/AbstractTemplatorParse.php

```diff
@@ -6,18 +6,19 @@
 
 abstract class AbstractTemplatorParse
 {
-    protected string $templateDir;
+    protected TemplatorFinder $finder;
     protected string $cacheDir;
+
     /**
      * Uses poller.
      *
      * @var string[]
      */
     protected $uses = [];
 
-    final public function __construct(string $templateDir, string $cacheDir)
+    final public function __construct(TemplatorFinder $finder, string $cacheDir)
```

Comment:
Backwards compatibility please add string parameter support.

Reply:
we don't need this because is an internal.